longevity_ics_test.py

```diff
@@ -0,0 +1,40 @@
+from longevity_test import LongevityTest
+
+DICT_KEYSPACES_COMPRESSION = {'keyspace_lz4': 'LZ4', 'keyspace_deflate': 'Deflate', 'keyspace_snappy': 'Snappy'}
+DICT_DEFAULT_BLOB_COLUMNS = {'"C0"': 'blob', '"C1"': 'blob', '"C2"': 'blob', '"C3"': 'blob', '"C4"': 'blob'}
+
+
+class IcsLongevetyTest(LongevityTest):
+    def _pre_create_schema_with_compaction(self, keyspace_num=1, scylla_encryption_options=None, single_column=False,  # pylint: disable=too-many-arguments
+                                           compaction='IncrementalCompactionStrategy', compression=None,
+                                           create_all_keyspaces_table=True):  # pylint: disable=too-many-arguments
+        """
+        For cases we are testing many keyspaces and tables, It's a possibility that we will do it better and faster than
+        cassandra-stress.
+        """
+        columns = {'"C0"': 'blob'} if single_column else DICT_DEFAULT_BLOB_COLUMNS
```

Comment:
no need to support the "single column scenario"

Reply:
done